drivers/npmx/npmx_shell.c

```diff
@@ -240,7 +240,7 @@ static int cmd_charger_charging_current_set(const struct shell *shell, size_t ar
 	}
 
 	int err = 0;
-	uint16_t current = (uint16_t)shell_strtoul(argv[1], 0, &err);
+	uint16_t current = CLAMP(shell_strtoul(argv[1], 0, &err), 0, UINT16_MAX);
 
 	if (err != 0) {
 		shell_error(shell, "Error: current has to be an integer.");
```

Comment:
Here clamping can not be only about wring argument type. There should be another print with "wrong value" or "too big value"

Reply:
It's out of the scope of this PR, however for me this implementation is ok. The function `npmx_charger_charging_current_set()` returns error `NPMX_ERROR_INVALID_PARAM`, thus shell returns `Error: unable to set charging current value.`

Reply:
Will be fixed in:
 - https://github.com/NordicSemiconductor/npmx-zephyr/pull/40